assets/css/tasks/_contact-tasks.css

```diff
@@ -0,0 +1,98 @@
+/* CONTACT PAGE */
+
+/* assets/css/pages/_contact.css */
+.contact-section {
+    background-image: url(/img/contact-img.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: right 0rem bottom 0;
+   
+}
+.u-center-text {
+    text-align: center;
+}
+.form {
+    /* padding;  set the same space around the contents of the box */
```

Comment:
It is really nice that you use `rem` unit :) I think it is better than `px` or `em`. We can more control of the size of styled element be using `rem` because it depends on root font size.

Reply:
You can delete coments, becouse they aren't give us any more necessery information about styles, becouse they are in your code :)